iphone/cli/commands/_build.js

```diff
@@ -4535,7 +4542,13 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 		},
 
 		function createAssetImageSets() {
-			this.logger.info(__('Creating assets image set'));
+			if (this.useAppThinning) {
+				this.logger.info(__('Creating assets image set'));
+			} else {
+				this.logger.info(__('Skip creating assets image set'));
+				return;
```

Comment:
This is a bit hard to read. The "then" falls through, but the "else" returns. Please reorder this to be easier to follow. Also, I think it would be a good idea to state why we're skipping the creation of the asset image sets.

```javascript
if (!this.useAppThinning) {
    this.logger.info(__('App thinning disabled, skipping asset image sets'));
}
```